File: node/ibge_api.py
import http.client

conn = http.client.HTTPSConnection("servicodados.ibge.gov.br")


payload = ""

conn.request("GET", "/api/v1/localidades/estados?view=nivelado&orberBy=nome", payload)
response = conn.getresponse()
result = response.read()

print(result.decode("utf-8"))

File: node/ibgeAPI.js
async function apiTest() {

    let response = await fetch("https://servicodados.ibge.gov.br/api/v1/localidades/estados?view=nivelado&orberBy=nome", {
        method: "GET",
    });

    let data = await response.text();
    console.log(data);
}

apiTest()